src/build/smart-contracts.md

```diff
@@ -0,0 +1,339 @@
+---
+title: Smart Contracts
+---
+
+# Smart Contracts
+
+This article explains the following:
+
+* [Deployment and interaction of CW20](/build/smart-contracts.html#deployment-and-interaction-of-cw20)
+
+### Install rustup
+
+First, install [rustup](https://rustup.rs/). Once installed, make sure you have the wasm32 target. This is an important step before running any contract to ensure that you are running Rust 1.44.1+ with wasm32-unknown-unknown target installed.
+
+```
+rustup --version
+rustup default stable
+default stable
+cargo --version
+# If this is lower than 1.51.0+, update
```

Comment:
perhaps something like: "... please update using the commands below" or similar :)